tools/integration-test/src/tests/fee/register_payee.rs

```diff
@@ -0,0 +1,153 @@
+use ibc_relayer_types::core::ics04_channel::version::Version;
```

Comment:
I'm unclear on why the expected behavior of this test is what it is on account of there being calls to both `register_payee` and `register_counterparty_payee`. What is the purpose of calling both of these functions?

Reply:
From what I understand, the `register_payee` allows the user to specify the `payee` receiving the `ack_fee`. In the case where only `register_counterparty_payee` is called, let's say with address `relayer_a`, then the `recv_fee + ack_fee` will go to the to `relayer_a`. But if `register_payee` is called, with address `payee_a`,  and `register_counterparty_payee` is called with `relayer_a`, then `relayer_a` will only get `recv_fee` and `payee_a` will only get `ack_fee`.
One possible usage would be if the forward and reverse relayer are the same, but the relayer wishes to store the `recv_fee` and `ack_fee` in separate wallets.

Reply:
Makes sense, thanks for clearing that up! 🙂